### PR TITLE
Fold CompletionQueue and ServerCompletionQueue into grpc_impl

### PR DESCRIPTION
**Pull request by karthikravis**
Move CompletionQueue and ServerCompletionQueue classes into grpc_impl namespace from grpc namespace.

### CODE REVIEW

File: include/grpcpp/impl/codegen/server_interface.h.rej [hunk 1]
```diff
@@ -0,0 +1,19 @@
+--- include/grpcpp/impl/codegen/server_interface.h
```
Comment (mhaidrygoog) [inline include/grpcpp/impl/codegen/server_interface.h.rej]: Please remove this file

Comment (karthikravis) [reply include/grpcpp/impl/codegen/server_interface.h.rej]: Thanks for catching this! Removed now.

File: include/grpcpp/channel.h [hunk 2]
```diff
@@ -19,97 +19,19 @@
 #ifndef GRPCPP_CHANNEL_H
```
Comment (mhaidrygoog) [inline include/grpcpp/channel.h]: I think we need the copyright header here

Comment (mhaidrygoog) [reply include/grpcpp/channel.h]: nevermind

**Comment by karthikravis** [comment]
The remaining tests passed before a trivial rebase. Bypassing tests because of infra issues.

**Pull request merged**